api/office/schema.py

```diff
@@ -1,4 +1,6 @@
 import graphene
+# from flask.ext.mail import Mail, Message
+# from celery import Celery
 from graphene_sqlalchemy import SQLAlchemyObjectType
```

Comment:
@blac-siren Good job, You could however, remove the commented code at this point.